### ISSUE TITLE
"Custom link name" in link references break linking on import

### ISSUE THREAD

**Issue by MisterIXI** [Praxxian/lava-flow#15]
When using [custom link names](https://help.obsidian.md/How+to/Internal+link) in obsidian (e.g. `[[mdFile|customname]]`) the recognition for linking breaks.  
Meaning this: 
```
[[Ritual]]  
[[Ritual|test]]  
```
gets imported as 
```
@JournalEntry[pIF5BcBvcIDhvI1x]{Ritual}
[[Ritual|test]]
```

The best solution would be to store the custom name for use as a name in Foundry. Alternatively everything after the pipe could be ignored.  
  
I will see if I can fix this myself and submit a PR. Should be a fairly simple thing to fix/add.

**Comment by iamthetot** [comment]
Also having this issue. Was looking forward to using this importer, but this alone is unfortunately a deal-breaker. Will be eagerly watching for a fix!

**Comment by Praxxian** [comment]
Sorry all for the delay. Had some family stuff happen. I understand the issue. I will check this out.

**Comment by Praxxian** [comment]
This should be fixed in v1.1.2. Thanks for your feedback!
